--- /dev/null
+++ ../src/Database/Table.php
@@ -32,6 +32,7 @@
  */
 abstract class Table extends Base {
 
+	protected $site_id = null;
 	/**
 	 * Table name, without the global table prefix.
 	 *
@@ -206,7 +207,7 @@
 		}
 
 		// Update interface for switched site
-		$this->set_db_interface();
+		$this->set_db_interface($site_id);
 	}
 
 	/** Public Helpers ********************************************************/
@@ -824,8 +825,24 @@
 	 *
 	 * @since 1.0.0
 	 */
-	private function set_db_interface() {
+	private function set_db_interface($site_id = null) {
 
+		// Set variables for global tables
+		if ($this->is_global()) {
+			$site_id = 0;
+			$tables  = 'ms_global_tables';
+
+			// Set variables for per-site tables
+		} else {
+			$tables = 'tables';
+		}
+
+		if (null !== $site_id && $site_id === $this->site_id) {
+			return;
+		}
+
+		$this->site_id = $site_id;
+
 		// Get the database once, to avoid duplicate function calls
 		$db = $this->get_db();
 
@@ -832,17 +849,6 @@
 		// Bail if no database
 		if ( empty( $db ) ) {
 			return;
-		}
-
-		// Set variables for global tables
-		if ( $this->is_global() ) {
-			$site_id = 0;
-			$tables  = 'ms_global_tables';
-
-		// Set variables for per-site tables
-		} else {
-			$site_id = null;
-			$tables  = 'tables';
 		}
 
 		// Set the table prefix and prefix the table name
