--- /dev/null
+++ ../src/Broker/Cookies.php
@@ -43,7 +43,8 @@
     /**
      * @inheritDoc
      */
-    public function offsetSet($name, $value)
+	#[\ReturnTypeWillChange]
+	public function offsetSet($name, $value)
     {
         $success = setcookie($name, $value, time() + $this->ttl, $this->path, $this->domain, $this->secure, true);
 
@@ -66,7 +67,8 @@
     /**
      * @inheritDoc
      */
-    public function offsetGet($name)
+	#[\ReturnTypeWillChange]
+	public function offsetGet($name)
     {
         return $_COOKIE[$name] ?? null;
     }
@@ -74,7 +76,8 @@
     /**
      * @inheritDoc
      */
-    public function offsetExists($name)
+	#[\ReturnTypeWillChange]
+	public function offsetExists($name)
     {
         return isset($_COOKIE[$name]);
     }
